pkg/controllers/grafana-operator/controller.go

```diff
@@ -58,17 +56,18 @@ const (
 	operatorGroupName = "monitoring-stack-operator-grafana-operator"
 	grafanaName       = "monitoring-stack-operator-grafana"
 	grafanaCSV        = "grafana-operator.v4.1.0"
+
+	managedBy    = "app.kubernetes.io/managed-by"
+	operatorName = "monitoring-stack-operator"
 )
 
 type reconciler struct {
 	controller              controller.Controller
 	grafanaWatchEstablished bool
-	k8sClient               client.Client
+	cache                   cache.Cache
+	msoClient               client.Client
```

Comment:
Can we keep this as `k8sClient`?

Reply:
I think we should differentiate so that the `msoClient` means a client that makes use of the `cache`  with informers watching only the `monitoring-stack-operator` namespace as opposed to `k8sClient` that we use else where which has the `mgr.GetCache()` attached/associated.

Reply:
Hm, should we call it `nsClient` then? I suspect the namespace will change at one point, and mso is a bit ambiguous

Reply:
agreed! should we delete the k8sClient .. which is now unused ?

Reply:
actually .. I delete it already :D